docs/atomic-pull.1.md

```diff
@@ -59,6 +59,10 @@ It tells Skopeo to not do TLS verification on the specified registry.
 
 `atomic pull --storage ostree http:REGISTRY/IMAGE:TAG`
 
+Not fully qualified images are supported when pulling to 'ostree',
```

Comment:
Just to confirm, here the qualified image will be in the form of REGISTRY/IMAGE:TAG right? The reason why I aksed that was because ... https://github.com/projectatomic/atomic/pull/1124/files#diff-21fc15fd5e971e0a8aad60171d003206L26, the description there kinda confuses me.

Reply:
yes exactly, what we really want is that the registry is specified as well

Reply:
May be we want to be a slightly more specific here about **qualified name** then? It is really my personal opinion tho, I feel we can tell the user what fully qualified name is as it is not mentioned explicitly anywhere in this doc? Tho, both way feels ok to me. Other than this nit, LGTM :-).

Reply:
I have added a fixup patch, does it look better?

Reply:
yup, LGTM =)